src/lesson4/task1/List.kt

```diff
@@ -214,7 +226,26 @@ fun decimalFromString(str: String, base: Int): Int = TODO()
  * 90 = XC, 100 = C, 400 = CD, 500 = D, 900 = CM, 1000 = M.
  * Например: 23 = XXIII, 44 = XLIV, 100 = C
  */
-fun roman(n: Int): String = TODO()
+
+fun roman(n: Int): String {
+    val numbers = intArrayOf(1, 4, 5, 9, 10, 40, 50, 90, 100, 400, 500, 900, 1000, 5000)
+    val letters = arrayOf<String>("I", "IV", "V", "IX", "X", "XL", "L", "XC", "C", "CD", "D", "CM", "M", "MMMMM")
+    var romanValue = ""
+    var N = n
+    while (N > 0)
+    {
+        for (i in 0..numbers.size - 1)
+        {
+            if (N < numbers[i])
```

Comment:
Very suspicious comparison. Please run local tests to check this function correctness.